tools/basictool.py

```diff
@@ -0,0 +1,113 @@
+"""CLI tool for loopchain"""
+import inspect
+import os
+import traceback
+from collections import OrderedDict
+from pathlib import Path
+from sys import platform
+
+from earlgrey import MessageQueueService
+from tools import get_option_from_prompt
+
+from testcase.unittest.test_util import clean_up_temp_db_files, clean_up_mq
```

Comment:
Looks weird to use test helper located at test dir.

How about re-locate them?

Reply:
'basictool.py' is the same as the private version(=demotool.py). So I have no interest about refactoring now.

Reply:
The goal of this PR is to add a new single feature. And to make it open source. After then we will discuss about refactoring.